backend/ee/danswer/external_permissions/google_drive/doc_sync.py

```diff
@@ -27,6 +31,42 @@
 logger = setup_logger()
 
 
```

Comment:
Running the connector is now done on a source by source basis. 

This depends on the size of the api load that running the connector will cause.

For google drive, polling is supported, so the load isnt that large

Reply:
if a bunch of them need to run the connector, _get_docs_with_additional_info could get moved to the utils